lessons/07-html-forms/index.html

```diff
@@ -0,0 +1,38 @@
+<html>
+    <head>
+        <title>Create User</title>   
+    </head>
+    <body>
+        <h1>Create User</h1>
+        <hr/>
+        <form action="create-user.html" method="get">
+            <label>Email:</label>
+            <input type="email" name="email" placeholder="email" required/> <br>
+            <label>Username:</label>
+            <input type="text" name="username" placeholder="username" size="30" maxlength="12"/> <br>
+            <label>Password:</label>
+            <input type="password" name="password" placeholder="password" size="30" maxlength="8" required/> <br>
```

Comment:
Rather than separating these using a `<br />` we should group the label + input's together using `<div>` tags. We want to reserve using `<br />` for only inside of paragraphs

```html
<div>
  <label> Username:</label>
  <input type="text" name="username" placeholder="username" size="30" maxlength="12" />
</div>

<div>
  <label>Password:</label>
  <input type="password" name="password" placeholder="password" size="30" maxlength="8" required/>
</div>

...etc
```